tests/cli/main.cpp

```diff
@@ -387,4 +387,4 @@ BOOST_AUTO_TEST_CASE( cli_set_voting_proxy )
       throw;
    }
    app1->shutdown();
-}
+}
```

Comment:
Please undo this change.

Reply:
ok sure, thx !

Reply:
Please undo this change (you removed the `newline` at end of file).